src/common/index.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import Handsontable from 'hot-alias';
-import {SettingsMapper} from './settingsMapper';
+import { SettingsMapper } from './settingsMapper';
+
+export interface Props extends Handsontable.DefaultSettings {
+  data?: any[][] | object[];
+  id?: string,
+  className?: string,
+  style?: React.CSSProperties,
+  settings?: Handsontable.DefaultSettings
```

Comment:
Just for the sake of discussion. I've noticed that in angular wrapper we're using `GridSettings` which is basically the same as `DefaultSettings` but looks more like a type than an instance. `DefaultSettings` are "THE settings". Probably doesn't matter that much, but would be cool to be consistent across wrappers too. What do you think @jansiegel  @swistach ?

Reply:
If I'm reading this correctly, `DefaultSettings` extends `GridSettings` with... nothing, so they really are the same thing. But I think using `DefaultSettings` makes more sense, as it potentially _might_ be extended with something someday (and we will probably want to be able to make use of the new properties).